src/App.tsx

```diff
@@ -5,7 +5,21 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [isMovieNew, setIsMovieNew] = useState(true);
+
+  const addMovie = (movie: Movie) => {
+    const isExistMovie = movies.find(
+      existMovie => existMovie.imdbId === movie.imdbId,
```

Comment:
use decomposing structure

Reply:
I rather use without decomposing in this case because its make my code more readibly and understantable